Packages/cc.starlessnight.unity-jsb/Source/Binding/DynamicType.cs

```diff
@@ -210,6 +212,17 @@ public ClassDecl Bind(TypeRegister register, bool crossbind = false)
             }
             #endregion
 
+            if (_parentType != null)
+            {
+                var ctor = cls.GetConstructor();
+                var parentCtor = db.GetConstructorOf(_parentType.type);
+                JSApi.JS_SetPrototype(ctx, ctor, parentCtor);
+
+                var prot = db.GetPrototypeOf(type);
+                var parentProto = db.GetPrototypeOf(_parentType.type);
+                JSApi.JS_SetPrototype(ctx, prot, parentProto);
+            }

```

Comment:
Added this so that classes inherit their prototype from prototype of their BaseClass. A code similar to this exists in other place but that does not work for all cases. This seems like a more central place to solve this problem. I don't know if the other place should be kept.